src/chrome/content/rules/Enjin.xml

```diff
@@ -1,47 +1,64 @@
-
 <!--
-Disabled by https-everywhere-checker because:
-Fetch error: http://dracokraft.enjin.com/ => https://dracokraft.enjin.com/: Too many redirects while fetching 'https://dracokraft.enjin.com/'
+	IMPORTANT INFORMATION:
 
-	CDN buckets:
-		- s3.amazonaws.com/files.enjin.com/
+	Sites, developed by the public, created on the Enjin Network have the ability to force
+	HTTP or HTTPS themselves. If they enable HTTPS, the site will automatically force a
+	HTTPS connection. You cannot add a rule to force HTTPS for sites that specifically
+	request to force the HTTP protocol, doing so will cause a redirect loop
+	(between the HTTP and HTTPS protocols) to occur.
 
-		- d2umfjgc9kwn9b.cloudfront.net
-			- assets-cloud.enjin.com
+	This functionality may be altered in the near future to ensure all sites use HTTPS and proxy
+	insecure material (such as scripts, stylesheets and images) through a HTTPS-enabled proxy.
 -->
-<ruleset name="Enjin" platform="mixedcontent" default_off='failed ruleset test'>
+
+<ruleset name="Enjin" platform="mixedcontent">
```

Comment:
Are there any serious mixedcontent issues on the sites? I didn't found any so far.

Reply:
The user's are capable of adding their own content which may cause mixed content, the official website has a flag to automatically upgrade all of our resources to HTTPS so you won't see mixed content on our main website.